src/Clap-Core/ClapApplication.class.st

```diff
@@ -43,14 +43,20 @@ ClapApplication >> context: aClapContext [
 	context := aClapContext
 ]
 
+{ #category : #execution }
+ClapApplication >> errorStreamDo: aBlock [
+	aBlock value: self context stderr
+]
+
 { #category : #execution }
 ClapApplication >> execute [
 	self subclassResponsibility
 ]
 
 { #category : #execution }
 ClapApplication >> executeWithContext: aClapContext [
```

Comment:
I think executeWithContext: aClapContext could be reduced to just #executeWithContext (without any parameter) since sender of this set's up context in upper method (see #ClapApplication>>command method)

Reply:
@demarey Let me know, what do you think about this.